app/javascript/widget/router.js

```diff
@@ -8,12 +14,12 @@ const router = createRouter({
     {
       path: '/unread-messages',
       name: 'unread-messages',
-      component: () => import('./views/UnreadMessages.vue'),
+      component: UnreadMessages,
```

Comment:
Any reason why this was changed?

Reply:
@pranavrajs No particular reason, I took it from the branch you worked on [feat/widget-v2](https://github.com/chatwoot/chatwoot/pull/11016/files#diff-89ef130de6d1bd8967c9eb374df55d675040c3cefab5bdfc90a093688d13a111).